### DISCUSSION TITLE
Add option to generate without minification

### DISCUSSION THREAD

**Discussion by wahyudies** [luangjokaj/fuzzymail#8] [Ideas]
Hello,
Could you like to add new feature, like options to generate email file without minify the html code. Because may some other developer need readable html code for development purpose only.
Also add new plugin like `gulp-beautify-code` if html minify disable, because indentation in text editor (after build) a little bit messy.

Thanks! 🙏

**Comment by luangjokaj** [comment]
This is a good idea. I find myself doing this again and again. This could be done in two different ways:
1. Add a parameter for the production command, something like `--beautify` or `--disable-html-min`
2. Add support for configuration `fuzzymail.config.js`
